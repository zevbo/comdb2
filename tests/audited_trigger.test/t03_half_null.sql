CREATE TABLE t(i int, s char(65))$$

CREATE LUA TRIGGER shoutout_to_monse ON (TABLE t FOR INSERT) AUDITED 

INSERT INTO t values(4)
INSERT INTO t values(5, "abc")

SELECT * FROM "$audit_t"

DROP TABLE t
DROP TABLE "$audit_t"
DROP LUA TRIGGER shoutout_to_monse