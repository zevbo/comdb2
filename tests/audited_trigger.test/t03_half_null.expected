[CREATE TABLE t(i int, s char(65))] rc 0
[CREATE LUA TRIGGER shoutout_to_monse ON (TABLE t FOR INSERT) AUDITED] rc 0
(rows inserted=1)
[INSERT INTO t values(5, "abc")] rc 0
[SELECT * FROM "$audit_t"] rc 0
[DROP TABLE t] rc 0
[DROP TABLE "$audit_t"] rc 0
[DROP LUA TRIGGER shoutout_to_monse] rc 0
